objects-api/src/main/kotlin/com/ritense/objectsapi/listener/ObjectsApiFormDataListener.kt

```diff
@@ -0,0 +1,13 @@
+package com.ritense.objectsapi.listener
+
+import com.ritense.document.exception.NotImplementedException
+import com.ritense.valtimo.contract.event.ExternalDataSubmittedEvent
+import org.springframework.context.event.EventListener
+
+class ObjectsApiFormDataListener {
+
+    @EventListener(ExternalDataSubmittedEvent::class)
+    fun handle(event: ExternalDataSubmittedEvent) {
+        throw NotImplementedException()
```

Comment:
This will throw exceptions every time an ExternalDataSubmittedEvent is published. I think this can be considered a breaking change. Maybe log something instead?